src/site/content/en/blog/generic-sensor/index.md

```diff
@@ -0,0 +1,523 @@
+[`AbsoluteOrientation`](https://www.w3.org/TR/orientation-sensor/#absoluteorientationsensor) sensor
+provides a ready-to-use four-by-four rotation matrix based on the data obtained from the
+accelerometer, gyroscope, and magnetometer.
+
+You might think that the web platform already provides sensor data and you are absolutely right! For
+instance, [`DeviceMotion`](https://developer.mozilla.org/en-US/docs/Web/API/DeviceMotionEvent) and
+[`DeviceOrientation`](https://developer.mozilla.org/en-US/docs/Web/API/DeviceOrientationEvent)
+events expose motion sensor data. So why do we need a new API?
+
+Comparing to the existing interfaces, Generic Sensor API provides a great number of advantages:
+
+- Generic Sensor API is a sensor framework that can be easily extended with new sensor classes and
+  each of these classes will keep the generic interface. The client code written for one sensor type
+  can be reused for another one with very few modifications!
+- You can configure the sensor. For example, you can set the sampling frequency suitable for your
+  application needs.
+- You can detect whether a sensor is available on the platform.
+- Sensor readings have high precision timestamps, enabling better synchronization with other
+  activities in your application.
+- Sensor data models and coordinate systems are clearly defined, allowing browser vendors to
+  implement interoperable solutions.
+- The Generic Sensor based interfaces are not bound to the DOM (meaning they are neither `navigator`
+  nor `window` objects), and this opens up future opportunities for using the API within service
+  workers or implementing it in headless JavaScript runtimes, such as embedded
+  devices.
+- [Security and privacy](#privacy-and-security) aspects are the top priority for the Generic Sensor
+  API and provide much better security as compared to older sensor APIs. There is integration with
+  the Permissions API.
+- Automatic [synchronization with screen coordinates](#synchronization-with-screen-coordinates) is
+  available for `Accelerometer`, `Gyroscope`, `LinearAccelerationSensor`,
+  `AbsoluteOrientationSensor`, `RelativeOrientationSensor` and `Magnetometer`.
+
+## Browser compatibility
+
+The Generic Sensor API is supported by Google Chrome as of version 67. Most Chromium-derived
+browsers like Microsoft Edge, Opera, or Samsung Internet support this API as well. For other
+browsers, see [Can I use](https://caniuse.com/mdn-api_sensor). Note that the Generic Sensor API can
+be [polyfilled](#polyfill).
+
+## Available generic sensor APIs {: #available-generic-sensor-apis }
+
+At the time of writing, there are several sensors that you can experiment with.
+
+**Motion sensors:**
+
+- `Accelerometer`
+- `Gyroscope`
+- `LinearAccelerationSensor`
+- `AbsoluteOrientationSensor`
+- `RelativeOrientationSensor`
+- `GravitySensor`
+
+**Environmental sensors:**
+
+- `AmbientLightSensor` (Behind the `#enable-generic-sensor-extra-classes` flag.)
```

Comment:
Note that this applies only to Chromium-based browsers

Reply:
Added "in Chromium".